src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -10,9 +10,12 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU1605</NoWarn>
```

Comment:
What is the downgrade warning that occurs in this case?

Reply:
Hmm.. I started to get nuget downgrade errors about 'Microsoft.Rest.ClientRuntime.Azure' being 3.3.19 in StorageSync.Test compared to some higher version in use by StorageSync's autogenerated SDK, after I updated dependencies to newer SDK. I can't repro this error now, though. Will give it a try with warning suppression removed.

Reply:
`msbuild` at project root succeeds without suppression; removed.